Resources/Prototypes/_NF/Recipes/Reactions/medicine.yml

```diff
@@ -8,4 +8,18 @@
     Lipozine:
       amount: 1
   products:
-    Tramoxadone: 2
+    Traumoxadone: 2
+    
+- type: reaction
+  id: Stelloxadone
+  reactants:
+    Cryoxadone:
+      amount: 3
+    Stellibinin:
+      amount: 5
+    Arithrazine:
+      amount: 2
+  products:
+    Stelloxadone: 5
+    Water: 3
+    Fiber: 2
```

Comment:
plants have a lot of water and fiber in them.  this was so that rather than saying if i mix a volume of 10 units together i only get 5 units out, i have 10 units and i get out 10 units.  just only 5 of those units were of what you wanted the other 5 are "waste materials" such as the water and the fiber that were in the plants chemicals but have now been removed via reaction.  i could probably be sold on it being something else but if i remember correctly the only thing you get out of galaxy thistle is the stellibinin which means the water and fibers are still in that mixture of plant chemical.  i figure balancing will be happening over time to find a good feal for it as its played